src/main/java/uk/gov/digital/ho/hocs/aws/SqsMessageListenerContainerFactory.java

```diff
@@ -19,6 +19,23 @@ public SimpleMessageListenerContainerFactory awsMessageListenerContainerFactory(
         return createMessageFactory(amazonSqs, maxMessages, null);
     }
 
+    @Primary
```

Comment:
You shouldn't need this queue - so these beans can go

Reply:
Removed.